### DISCUSSION TITLE
Any way to update a project with just the dependencies available in the store?

### DISCUSSION THREAD

**Discussion by josh-hemphill** [pnpm/pnpm#7422] [Q&A]
I have a flaky proxy I have to run through, and my updates keep getting stuck, but it does manage to download some of the dependencies. Is there any way to update offline with just what's in the store so I can at least update what it was able to download so far?

**Comment by zkochan** [comment]
No. We have the `prefer-offline` option but we don't currently automatically switch to offline if requests fail.
